external/storm-jdbc/src/main/java/org/apache/storm/jdbc/common/Column.java

```diff
@@ -0,0 +1,87 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.storm.jdbc.common;
+
+
+import java.io.Serializable;
+import java.lang.reflect.Field;
+import java.sql.Date;
+import java.sql.Time;
+import java.sql.Timestamp;
+import java.sql.Types;
+
+public class Column<T> implements Serializable {
+
+    private String columnName;
+    private T val;
+    private int sqlType;
+
+    public Column(String columnName, T val, int sqlType) {
```

Comment:
I have added a javadoc that should help to clarify things. If we consider a database table as a list of rows and each row as a list of columns, this class represent a single instance of a column, so for a row [userID =1 , userName = "Foo"] we will have two instances of Column class
new Column("userId", 1, Types.INTEGER) and new Column("userName", "Foo", Types.Varchar). 

I am not sure why java's jdbc API decided to go with integer to represent sql Types instead of using more readable Enums but the sqlType has value mapped to java.sql.Types where each constant represents a database type.